SCClassLibrary/Common/Audio/UGenResourceManagers.sc

```diff
@@ -0,0 +1,131 @@
+UGenResourceManager {
+	classvar allResourceManagers;
+	*register { |resourceManager| allResourceManagers = allResourceManagers.add(resourceManager) }
+	*createNewInstances { ^allResourceManagers.collect({|m| m -> m.new }).asEvent }
+
+	add { |ugen| this.subclassResponsibility(thisMethod) }
+	panic { |ugen| this.subclassResponsibility(thisMethod) }
+	getAll { this.subclassResponsibility(thisMethod) }
```

Comment:
From the use below it seems as if `all` could just have a getter? Not sure I may have missed something.